### DISCUSSION TITLE
Please merge oauth and email login if they have the same email

### DISCUSSION THREAD

**Discussion by benwoodward** [deepgram/community#211] [General help] [answered]
I have two accounts with the same email, one was an email address sign up, and the other was logging in with Github. Can never remember which is the right one. Would be great if you merged the accounts automatically if they have the same email address.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi Ben, we purposely do not merge Deepgram accounts that happen to have the same email address, because the underlying data may differ later - for instance, you may log in with email and also through OAuth with a GitHub account with the same email, but later you change the email address associated with your GitHub account.

I understand that it's difficult to juggle two different accounts though - my suggestion is to share projects between the accounts if you do not have a need for the account separation, and you don't want to simply delete one account.

1. From Deepgram account A, select the project you want to share using the project toggle menu at the top of the page.
2. Go to Settings (left-hand side) -> Team Members -> Invite New Member.
3. Send an invite to your email address.
4. Log out of account A and log into account B (or use a private browser).
5. While logged into account B, accept the email invite and join the shared project.

This will allow you to access the project from both accounts.

To add further clarity, you can go into Settings -> Project Settings, and rename each project to something more descriptive than "<email>'s Project". Additionally, in the top right corner, you can go into Profile Settings and give each account a different FirstName/LastName. I hope that some of these ideas will help you manage your accounts more easily!

**Reply by benwoodward** [reply]
ok thanks
